README.md

```diff
@@ -2,38 +2,28 @@
 
-## Running
+## Running Locally
 
-The quickest way to get the project running is to use the dev container that has been configured to install all
-dependencies required to run the project locally. To use the dev container, see the [Data Platform docs.](https://technical-documentation.data-platform.service.justice.gov.uk/documentation/platform/infrastructure/developing.html#developing-the-data-platform)
+The dashboard is run in a DevContainer via Docker. The DevContainer VSCode extension is recommended, as is Docker Desktop.
 
-Alternatively you can install the project locally by installing python 3.11, creating a venv, and installing the
-project dependencies with
+For more information on Dev Containers, see the [Data Platform docs.](https://technical-documentation.data-platform.service.justice.gov.uk/documentation/platform/infrastructure/developing.html#developing-the-data-platform)
 
-```commandline
-pip install -r requirements.dev.txt
-```
 
-You will also need to have Postgresql installed and running on your machine.
+### Building the DevContainer
+To build the dev container, ensure docker desktop is running, then open the AP Dashboard project in VSCode. Open the command pallet by hitting command+shift+p and search for ```Dev Containers: Reopen in container``` and hit enter. This will build the dev container.
 
-### Pre-commit
+If you are using a workspace with multiple applications, search for ```Dev Containers: Open folder in Container…``` instead, then select the AP UI folder. Once the dev container has finished building, it should install all the required Python and npm dependencies, as well as run the migrations.
 
-To avoid pushing code and seeing the GitHub actions fail due to linting errors, when installing the project for the
-first time you should install the pre-commit hooks with:
+### Environment Variables
+There is an example environment file stored on 1Password named ```Analytical Platform Dashboard Env```. Paste the contents into a new file called ```.env``` in the root of the project.
 
-```commandline
-pre-commit install
-```
+### Running Development Server
+To run the server, you will need to use aws-sso cli. To find the correct profile, run ```aws-sso list``` in the terminal. This will provide you with a link to sign in via SSO. Once signed in, a list of profiles will be displayed. You are looking for the profile name linked to the ```analytical-platform-development``` AccountAlias.
```

Comment:
We'll need to change this when we switch to a new account, but is good for now

Reply:
That's fine